src/Core/Utility/VersionUtility.cs

```diff
@@ -664,6 +678,27 @@ public static string GetShortFrameworkName(FrameworkName frameworkName)
                 // only show version part if it's > 0.0.0.0
                 if (frameworkName.Version > new Version())
                 {
+                    if (frameworkName.Identifier.Equals(NetStandardAppFrameworkIdentifier, StringComparison.OrdinalIgnoreCase)
+                        || frameworkName.Identifier.Equals(NetStandardFrameworkIdentifier, StringComparison.OrdinalIgnoreCase)
+                        || frameworkName.Identifier.Equals(NetPlatformFrameworkIdentifier, StringComparison.OrdinalIgnoreCase))
+                    {
+                        if (frameworkName.Version.Major > 9

```

Comment:
Yep, fixed in https://github.com/NuGet/NuGet2/commit/885e64d38bf4d070309049e0724e1a38b0878d8e